### PR TITLE
FIX: Fix for Spyder

### PR DESCRIPTION
**Pull request by larsoner**
Give this a shot @mdclarke

### CODE REVIEW

File: mnefun/_mnefun.py [hunk 1]
```diff
@@ -685,7 +687,7 @@ def fetch_raw_files(p, subjects, run_indices):
             cmd += ['--include', op.basename(fname)]
         remote_loc = '%s:%s' % (p.acq_ssh, op.join(remote_dir, ''))
         cmd += ['--exclude', '*', remote_loc, op.join(raw_dir, '')]
-        run_subprocess(cmd)
+        run_subprocess(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         # move files to root raw_dir
```
Comment (ktavabi) [inline mnefun/_mnefun.py]: [Python pages](https://docs.python.org/2/library/subprocess.html#using-the-subprocess-module) recommends 

> Do not use stdout=PIPE or stderr=PIPE with this function as that can deadlock based on the child process output volume. Use Popen with the communicate() method when you need pipes.

You thinking this use case is simple enough to ignore this?

Comment (larsoner) [reply mnefun/_mnefun.py]: I think it should be okay

**Comment by larsoner** [comment]
@mdclarke reported that this seems to work so I'll merge, but let me know if there are other problems

**Comment by ktavabi** [comment]
Also resolves `pykernel` issue with `subprocess` in pycharm. Changes LGTM aside from comment/question about stdout=PIPE or stderr=PIPE usage.

**Pull request merged**